switch_model/solve.py

```diff
@@ -25,12 +25,7 @@
     create_model, _ArgumentParser, StepTimer, make_iterable, LogOutput, warn
 )
 from switch_model.upgrade import do_inputs_need_upgrade, upgrade_inputs
-
-#paty's adition for debugging:
-try:
-    from IPython import embed
-except:
-    pass
+import pdb
```

Comment:
Shall we remove this line and instead just use breakpoint(). If we decide to do that, we need to increase the python version required to >3.7. Which is not unreasonable.

Reply:
Yes, I didn't know that was an option, that's a lot better. I've just made the change.